packages/client/src/modals/Transaction/BannerInfo.js

```diff
@@ -0,0 +1,86 @@
+import React, { useEffect, useContext, useState } from "react";
+import { Web3Context } from "contexts/Web3";
+import { ACTION_TYPES } from "constants/enums";
+import Svg from "library/Svg";
+import { formatAddress, getTokenMeta, parseIdentifier, getNameByAddress } from "utils";
+
+const BannerInfo = ({ className = "", actionData = {}, contacts = {}, signers = [] }) => {
+  const { getAccountNFTReference } = useContext(Web3Context);
+  const [image, setImage] = useState();
+  const { recipient, nftId, collectionId, vaultId, tokenAmount, signerAddr } = actionData;
+  const { displayName, tokenType } = getTokenMeta(vaultId);
+  const { contractName: NFTName, contractAddress: NFTAddress } = parseIdentifier(collectionId);
+  const { name: imageName, imageURI } = image || {};
+  const actionType = actionData.type;
+
+  useEffect(() => {
+    if (actionType === ACTION_TYPES.TRANSFER_NFT) {
+      const getImageURL = async () => {
+        const result = await getAccountNFTReference(
+          NFTName,
+          formatAddress(NFTAddress),
+          recipient,
+          nftId
+        );
+        setImage(result);
+      };
+      getImageURL().catch(console.error);
+    }
+  }, [actionType, NFTName, NFTAddress, nftId, recipient, getAccountNFTReference]);
+
+  const contactName = getNameByAddress(contacts, signerAddr);
+
+  return (
+    <div className={className}>
+      {actionType === ACTION_TYPES.TRANSFER_NFT && (
+        <>
+          {imageURI && (
+            <img
+              className="columns is-vcentered is-multiline is-mobile mr-2 mt-2 success-modal-image"

```

Comment:
columns class should be attached to a container dom but this img doesn't have any child, what is the effect we  are trying to achieve here?

Reply:
I copy/pasted this code from https://github.com/DapperCollectives/VESSEL/blob/main/packages/client/src/modals/TransactionSuccessModal.js#L63-L79

was just trying to change as little as possible in moving stuff around